go/exec/status.go

```diff
@@ -18,9 +18,46 @@
 
 package exec
 
+import (
+	"github.com/vitessio/arewefastyet/go/storage"
+)
+
 const (
 	StatusCreated  = "created"
 	StatusStarted  = "started"
 	StatusFailed   = "failed"
 	StatusFinished = "finished"
 )
+
+type benchmarkStats struct {
+	Benchmark_total    int
+	Benchmark_finished int
+	Benchmark_30Days   int
+}
```

Comment:
The structure `benchmarkStats` should be exported as it is used as a return parameter of `GetBenchmarkStats` which is itself an exported function.

Moreover, the fields should follow the camelcase naming convention without underscores.